tests/rptest/scale_tests/many_topics_test.py

```diff
@@ -203,6 +222,65 @@ def acked(err, msg):
 
         return next_topic_batch, new_topic_names, errors
 
+    def _consume_messages_from_topics(self,
+                                      kclient,
+                                      message_count,
+                                      topic_names,
+                                      timeout_sec=3000):
+        """Creates a single consumer that will try to fetch all topics in 
+        `topic_names` in a single request
+
+        Args:
+            kclient (_type_): python_librdkafka client
+            message_count (_type_): number of messages to consume
+            topic_names (_type_): list of topic names to consume from
+            timeout_sec (int, optional): Timeout. Defaults to 300.
+        """
+        consumer_extra_config = {
+            "auto.offset.reset": "earliest",
+            "group.id": "topic_swarm_group_1",
+            "message.max.bytes": 1000000000,
+            "receive.message.max.bytes": 2147483647,
```

Comment:
what's the idea behind these two config overrides?

Reply:
Nice catch, this is just a remnant from debugging an issue. Removing the overrides.